src/main/java/com/github/wovnio/wovnjava/PathUrlLanguagePatternHandler.java

```diff
@@ -47,10 +47,10 @@ private Pattern buildGetLangPattern(String sitePrefixPath) {
     private Pattern buildRemoveLangPattern(String lang) {
         Pattern p = Pattern.compile(
                 "^(.*://)?" + /* optional schema */
-                "([^/]*)?" + /* optional host */
-                "(" + this.sitePrefixPath + "/)" + /* sitePrefixPath */
-                "(" + lang + ")" + /* lang code */
-                "(/|$)"
+                "([^/?]*)?" + /* optional host */
+                "(" + this.sitePrefixPath + ")" +
+                "(/" + lang + ")" +
+                "(/|\\?|$)" /* next path, query, or end-of-string */
```

Comment:
I think there is a possibility that `#` will continue.

Reply:
Would be possible if we were working in the widget. But the server will not handle fragments, so fortunately we don't have to worry about it.